### PR TITLE
Word wrap docstrings at 92-characters

### PR DESCRIPTION
**Pull request by omus**
As I was looking through base as part of #21699 I found some docstrings that were not word wrapped to 92-characters.

### CODE REVIEW

File: base/distributed/managers.jl [hunk 1]
```diff
@@ -58,48 +58,47 @@ location on each node, or to be available via a shared file system.
...
 * `exeflags`: additional flags passed to the worker processes.
 
-* `topology`: Specifies how the workers connect to each other. Sending a message
-            between unconnected workers results in an error.
+* `topology`: Specifies how the workers connect to each other. Sending a message between
+  unconnected workers results in an error.
 
-  + `topology=:all_to_all`  :  All processes are connected to each other.
-                      This is the default.
+    + `topology=:all_to_all`: All processes are connected to each other. The default.
```
Comment (omus) [inline base/distributed/managers.jl]: Changed the wording from "This is the default" to "The default" to fit better

File: base/process.jl [hunk 2]
```diff
@@ -31,42 +31,37 @@ struct Cmd <: AbstractCmd
...
+  `windows_verbatim=true` is useful for launching programs that parse their command line in
+  nonstandard ways.) Has no effect on non-Windows systems.
+* `windows_hide::Bool`: If `true` (defaults to `false`), then on Windows no new console
+  window is displayed when the `Cmd` is executed. This has no effect if a console is
+  already open or on non-Windows systems.
+* `env`: Set environment variables to use when running the `Cmd`. `env` is either a
+  dictionary mapping strings to strings, an array of strings of the form `"var=val"`, an
+  array or tuple of `"var"=>val` pairs, or `nothing`. In order to modify (rather than
+  replace) the existing environment, create `env` by `copy(ENV)` and then set
+  `env["var"]=val` as desired.
```
Comment (omus) [inline base/process.jl]: In addition to modifying the wrapping I removed several double spaces after a period.

File: base/distributed/managers.jl [hunk 3]
```diff
@@ -58,48 +58,47 @@ location on each node, or to be available via a shared file system.
...
+`port`.
 
-`count` is the number of workers to be launched on the specified host. If specified as `:auto`
-it will launch as many workers as the number of cores on the specific host.
+`count` is the number of workers to be launched on the specified host. If specified as
+`:auto` it will launch as many workers as the number of cores on the specific host.
 
 Keyword arguments:
 
 * `tunnel`: if `true` then SSH tunneling will be used to connect to the worker from the
```
Comment (Sacha0) [inline base/distributed/managers.jl]: Perhaps nix the leading extra space as well?

**Review by Sacha0 — APPROVED** [review]
Thanks @omus! :)

**Review by StefanKarpinski — APPROVED** [review]

**Comment by Sacha0** [comment]
(Absent objections, requests for time, or someone beating me to it, I plan to merge this tomorrow morning. Best!)

**Pull request merged**